ui/components/multichain/global-menu/global-menu.js

```diff
@@ -114,44 +106,22 @@ export const GlobalMenu = ({ closeMenu, anchorElement }) => {
 
       {
         ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
-        <>
-          {mmiPortfolioEnabled && (
-            <MenuItem
-              iconName={IconName.Diagram}
-              onClick={() => {
-                trackEvent({
-                  category: MetaMetricsEventCategory.Navigation,
-                  event: MetaMetricsEventName.UserClickedPortfolioButton,
-                });
-                window.open(mmiPortfolioUrl, '_blank');
-                closeMenu();
-              }}
-              data-testid="global-menu-mmi-portfolio"
-            >
-              {t('portfolioDashboard')}
-            </MenuItem>
-          )}
-
+        mmiPortfolioEnabled && (
           <MenuItem
-            iconName={IconName.Compliance}
+            iconName={IconName.Diagram}
             onClick={() => {
               trackEvent({
                 category: MetaMetricsEventCategory.Navigation,
-                event: MetaMetricsEventName.UserClickedCompliance,
+                event: MetaMetricsEventName.UserClickedPortfolioButton,
               });
-              if (getEnvironmentType() === ENVIRONMENT_TYPE_POPUP) {
-                global.platform.openExtensionInBrowser(
-                  COMPLIANCE_FEATURE_ROUTE,
-                );
-              } else {
-                history.push(COMPLIANCE_FEATURE_ROUTE);
-              }
+              window.open(mmiPortfolioUrl, '_blank');
+              closeMenu();
             }}
-            data-testid="global-menu-mmi-compliance"
+            data-testid="global-menu-mmi-portfolio"
           >
-            {t('compliance')}
```

Comment:
Just did a global search and this key/value is not user in other places. perhaps we can also do clean up in language file to remove it?

Reply:
You're right, removed it!